### ISSUE TITLE
Children demo seems to be broken

### ISSUE THREAD

**Issue by mypark** [valor-software/ng2-select#922]
Nothing shows up in the input of select a city by country - children example

https://valor-software.com/ng2-select/

**Comment by optimistex** [comment]
There is all fixes: https://github.com/optimistex/ng2-select-ex
There are demo: https://optimistex.github.io/ng2-select-ex

**Comment by optimistex** [comment]
Look at the pull request: #921

**Comment by javac9** [comment]
Yes, it seems to be broken. @optimistex , what you posted seems to be a fork of this project, right?

**Comment by optimistex** [comment]
Yes. I did a fork of the project and completely rewrote that.
The fork: https://github.com/optimistex/ng2-select-ex
Today I did release a stable version. Try it.
